idt_config/linux/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+SOURCE_PATH=$1
+IDE_PATH=<PATH TO IDE e.g. /Applications/MCUXpressoIDE_x.y.z>
+TOOLCHAIN_PATH=$IDE_PATH/ide/tools/bin
+IDE=$IDE_PATH/ide/MCUXpressoIDE.app/Contents/MacOS/mcuxpressoide
```

Comment:
This path looks MacOS specific, has this been tested on Linux to know what the path should be? I would expect the application structure to be different

Reply:
Makes sense. I think it might be a good compromise to replace the path after $IDE_PATH with <PATH TO IDE EXECUTABLE> so that a customer doesn't assume it's correct for their system

Reply:
looked into linux installation, the default installation is through a .deb that puts it in /usr/local so i made that the new recommended IDE_PATH